pkg/controller/release/strategy_executor.go

```diff
@@ -32,14 +32,15 @@ func (p *Pipeline) Enqueue(step PipelineStep) {
 }
 
 type Extra struct {
-	IsLastStep   bool
-	HasIncumbent bool
+	IsLastStep bool
+	HasTail    bool
+	Initiator  *shipper.Release
```

Comment:
I get what the Initiator is after reading the code, but without being familiar with how things work I don't think that the name carries enough meaning. I'm sorry I don't have a better suggestion, so I'll still approve, and if a better name comes up we can have a PR just to rename this.